daemon/emer-daemon.c

```diff
@@ -596,6 +699,18 @@ add_stored_events_to_builders (EmerDaemon        *self,
       return TRUE;
     }
 
+  if (has_invalid)
+    {
+      GVariant *payload = g_variant_new ("(tt)", num_variants, token);
+      g_variant_ref_sink (payload);
```

Comment:
I don't think this is needed... `emer_daemon_record_singular_event()` will consume the floating ref